user.py

```diff
@@ -0,0 +1,12 @@
+import abc
```

Comment:
Also, can we have a naming convention for interfaces like interface_user.py or user_interface.py just so we can distinguish them from each other?